### DISCUSSION TITLE
Copilot issues

### DISCUSSION THREAD

**Discussion by Knc37** [community/community#43444] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

So i have been trying to use copilot for months on platforms like Webstorm or IntelliJ but no success has been made till date.

everytime i tap on to the copilote icon, it gives me a code and tells me to log into my account, then they say the connection is successfull but i still can't use it in the platforms , cause there it shows i'm not connected. What should i do please?

![Screenshot_20230106_060203](https://user-images.githubusercontent.com/116761400/210934103-50d0505e-2915-4cd4-a6de-b50804fc248e.png)
![Screenshot_20230106_060314](https://user-images.githubusercontent.com/116761400/210934104-1715bd84-bdbf-4693-89c0-133d68cc57d6.png)

**Comment by Akash1134** [comment, marked as answer]
Hi @Knc37 , Copilot is not connected/signed in and it might be because of one or more of the reasons mentioned in the prerequisites [here](https://docs.github.com/en/copilot/quickstart#prerequisites). Also, were you prompted to sign in to Copilot on your IDE?

**Reply by Knc37** [reply]
Hi @Akash1134 , i already signed in with my github on my IDE (Webstorm) but still, i followed the steps shown on the link you sent me , but then i will be billed.  So how can i prove i´m a student on github to freely have access to copilot?

**Reply by iamcjwilliams** [reply]
@Knc37 Visit here to verify your student access: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#expiration-and-renewals

**Comment by jhonatanrojas** [comment]
I can't use copilot in vscode it shows me this message "You are not signed in to GitHub. Please sign in to use Copilot."

**Reply by iamcjwilliams** [reply]
@jhonatanrojas please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot
